content/en/docs/distributions/gke/customizing-gke.md

```diff
@@ -200,19 +160,53 @@ command:
 gcloud compute accelerator-types list
 ```
  
-To disable node-autoprovisioning, run `kfctl build` as described above.
-Then edit `${KF_DIR}/gcp_config/cluster-kubeflow.yaml` and set 
+To disable node-autoprovisioning, edit content for `${KF_DIR}/common/cluster/upstream/cluster.yaml`, set 
 [`enabled`](https://github.com/kubeflow/manifests/blob/4d2939d6c1a5fd862610382fde130cad33bfef75/gcp/deployment_manager_configs/cluster-kubeflow.yaml#L73) 
 to `false`:
 
 ```
     ...
-    gpu-type: nvidia-tesla-k80
-    autoprovisioning-config:
+    clusterAutoscaling:
       enabled: false
+      autoProvisioningDefaults:
     ...
 ```
 
+Then create the [ContainerNodePool](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool) resource adopting GPU, for exmaple:
+
+```
+  apiVersion: container.cnrm.cloud.google.com/v1beta1
+  kind: ContainerNodePool
+  metadata:
+    labels:
+      mesh_id: "proj-PROJECT_NUMBER" # {"$kpt-set":"asm-mesh-id"}
+    name: containernodepool-gpu
+  spec:
+    location: LOCATION # {"$kpt-set":"location"}
+    initialNodeCount: 1
+    autoscaling:
+      minNodeCount: 1
+      maxNodeCount: 5
+    nodeConfig:
+      machineType: n1-standard-4
+      diskSizeGb: 100
+      diskType: pd-standard
+      preemptible: false
+      oauthScopes:
+        - "https://www.googleapis.com/auth/logging.write"
+        - "https://www.googleapis.com/auth/monitoring"
+      guestAccelerator:
+        - type: "nvidia-tesla-k80"
+          count: 1
+      metadata:
+        disable-legacy-endpoints: "true"
+    management:
+      autoRepair: true
+      autoUpgrade: true
+    clusterRef:
+      name: "PROJECT/LOCATION/KUBEFLOW-NAME" # {"$kpt-set":"cluster-name"}
+```
+
```

Comment:
there's another outdated section "Add GPU node pool to an existing kubeflow cluster" below, remove it?

note, there's a step of installing nvidia driver to each node, you should keep it here.

Reply:
Sounds good, updated. 

Should I keep the "Add GPU node pool to an existing kubeflow cluster" for commandline instruction?

Reply:
You can decide, I probably prefer removing it -- to make doc consistent, always use cnrm resources to deployo

Reply:
Sounds good, I have removed the CLI instruction to recommend resource flie approach.